scripts/bundleSize/getBundleData.js

```diff
@@ -1,13 +1,11 @@
-const fs = require('fs');
-const { extractBundlesForPageType } = require('./pageTypeBundleExtractor');
+import fs from 'fs';
+import { extractBundlesForPageType } from './pageTypeBundleExtractor.js';
+import services from './serviceList.js'
```

Comment:
Moved away from using Cypress for service list as it necessitated converting much of Cypress to ESM which seemed excessive